### ISSUE TITLE
Tag more frequently 

### ISSUE THREAD

**Issue by LavaToaster** [jakubrohleder/angular-jsonapi#10]
I'm keeping up to date with changes with the packages, however in the past 7 days you've introduced some pretty cool features and fixed some restrictions, but they're untagged. To be able to use them I need to override my bower config to include the newest commit hash.

I suggest changing how you're currently versioning the alpha packages, as in my opinion it is a little awkward and less than semantic. Such as versioning under 1, using versions similar to 0.0.1 and such. So breaking changes result in 0.1.0 being released, and new features 0.1.1 and such.

I get that it's an alpha/in-dev package and you're not recommending anyone to use it in production _however_ this is the _only_ angular package that I've seen to properly support the spec and make it easy to use.

**Comment by jakubrohleder** [comment]
Hi @Lavoaster! 

I'm using http://semver.org/ convention for tagging and according to it x.x.x-alpha.x is a valid semantic tag. I jumped to 1.0.0-alpha straight away because I did not expect this stage to take me so long.

Nevertheless I got your point, today I had the same problem myself. I don't want to change previous tags, so what I propose is a rule: for alpha stage, that I'll make a new release (with tag) no longer then two days after introducing a new feature. 

Does it suit you?

Edit: I just did 1.0.0-alpha.3 release
